package/src/prmon.cpp

```diff
@@ -84,9 +92,29 @@ int ReadProcs(pid_t mother_pid, unsigned long values[4], unsigned long long valu
           fclose(file2);
 	}
 
+        snprintf(stat_buffer,64,"/proc/%llu/stat",(unsigned long long)*it);
+
+        FILE *file3 = fopen(stat_buffer,"r");
+
+        if(file3==0) {
+          openFails.push_back(std::string(stat_buffer));
+        }
+        else {
+          long clock_ticks = sysconf (_SC_CLK_TCK);
+          while(fgets(sbuffer,2048,file3)) {
+            tsbuffer = strchr (sbuffer, ')');
+            if(sscanf(tsbuffer + 2 , "%*c %*d %*d %*d %*d %*d %*u %*u %*u %*u %*u %80llu %80llu %80llu %80llu", &utime, &stime, &cutime, &cstime)) {
+              valuesCPU[0] = utime/clock_ticks;
+              valuesCPU[1] = stime/clock_ticks;
+              valuesCPU[2] = cutime/clock_ticks;
+              valuesCPU[3] = cstime/clock_ticks;
```

Comment:
I would not divide by `clock_ticks` here. We should keep resolution until we need to drop it (plus as your are doing integer division, you round down to the nearest second unnecessarily).